### PR TITLE
Typo - Fix #1896

### LINKED ISSUE #1896: Phenomizer broken on current master
Likely collateral damage from #1893.

### PR DESCRIPTION
**Pull request by dnil**
This PR  fixes a bug. Fix #1896.

**How to test**:
1. on stage, attempt to run phenomizer on a case with phenotypes. Current master will return no custom gene-list. This should.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN, CR

### CODE REVIEW

**Comment by dnil** [comment]
Ok, don't mend what ain't broken I guess. Still, it was an error waiting to happen. 

Now, this seems ok. Phenomizer adds genes:
Screenshot 2020-04-30 at 12 09 35

And single gene add from autocomplete still works:
Screenshot 2020-04-30 at 12 09 51

**Comment by dnil** [comment]
Ok, this appears ready for review.. 😆

**Comment by northwestwitch** [comment]
I'll review!

**Comment by northwestwitch** [comment]
Hi @dnil, could you describe a bit more how to trigger the bug. Phenomizer button works fine. Should I click on which button here. HPO panel?

**Comment by northwestwitch** [comment]
Never mind, I understood, from the Phenomizer page I have to select one of more phenotypes with genes

File: tests/server/blueprints/cases/test_cases_views.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def test_parse_raw_gene_symbols(app):
     """ Test parse gene symbols"""
 
     # GIVEN a list of autocompleted gene symbols
-    gene_symbols = ["MUTYH |POT1", "POT1| APC |PMS2"]
+    gene_symbols = ["MUTYH |POT1", "POT1 0.1|APC|PMS2"]
 
     # WHEN converting to hgnc_ids
     hgnc_symbols = parse_raw_gene_symbols(gene_symbols)
```
Comment (northwestwitch) [inline tests/server/blueprints/cases/test_cases_views.py:23]: Something I noticed but it was already there before this PR:
a couple of lines after this, there is this code:

```
# THEN the appropriate set of hgnc_ids should be returned
    assert hgnc_symbols == {"APC", "MUTYH", "PMS2", "POT1"}
```

Could you change, the comment, `hgnc_ids` to gene symbols?

**Pull request merged**